SalesforceCommerceCloud/pwa-kit
# Hide locale selector when there is only a single locale

# Description



Hide the locale selector when there is only a single supported locale, on footer and side drawer menu.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- hide selectors on footer and drawer menu

# How to Test-Drive This PR

- run `npm start`
- change supported locale list to only 1 locale
- check footer and menu that there is no selector

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
